README.md

```diff
@@ -116,25 +110,34 @@ In a different example, a service using the Functions Framework could have the u
+The framework **must** support at least the `http` and `cloudevents` function types. The framework **may** support legacy-style `event` function types. In statically typed languages, the framework must include an additional function type whereby the user can define unmarshalling rules. These function types dictate:
 
 - The steps taken by the framework in response to ingress requests
 - The function signatures which developers must adhere to when writing functions for use with a Functions Framework
 - The mechanism by which the developer signals that the function has completed performing useful work
 
 The developer must signal the function's signature type to the framework. This enables the framework to take appropriate unmarshalling, invocation and completion signalling steps.
 
-### HTTP Functions
+### HTTP Functions (Signature Type: `http`)
+
+The Functions Framework **must** support signature type `http`.
 
 When the container receives an ingress request, the framework must invoke the developer's function by passing a language-idiomatic representation of HTTP objects as arguments to the function. These objects must enable the developer to perform common HTTP tasks, such as inspecting the request's content encoding or headers. These objects should be accurate representations of the HTTP request received by the execution environment (i.e., path, body and headers should not be modified before passing them to the user's function).
 
-The developer's function must explicitly signal that it has completed performing useful work by sending a response to the Functions Framework. This response signals that the stateless container can be stopped.
+### CloudEvents Functions (Signature Type: `cloudevents`)
 
-### CloudEvents Functions
+The Functions Framework **must** support signature type `cloudevents`.
 
 When the container receives an ingress request, the framework must invoke the developer's function by passing an object corresponding to a [CloudEvents type](https://github.com/cloudevents/spec/blob/master/spec.md). This object does not expose HTTP semantics to the developer's function. The framework must handle unmarshalling HTTP requests into the CloudEvents object that is passed to the developer's function, and should support both [binary and structured content modes](https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#3-http-message-mapping) for incoming HTTP CloudEvent requests.
 
-The developer's function must either explicitly or implicitly signal that it has completed performing useful work. The function may explicitly signal this condition by explicitly returning. The function may implicitly signal this condition by simply evaluating until it reaches the end of the function's code block.
```

Comment:
Removing these sections about function termination seems out of scope? It seems like it may not be worded quite right so maybe it was just a pending change, but termination behavior seems like an important point to cover somewhere.

Reply:
Yeah, I think it should be kept, just not repeated 3 times.
Will re-add it.